client/tests/integration/triggers/time_trigger.rs

```diff
@@ -22,9 +22,9 @@ macro_rules! const_assert {
 /// But in general it works well and this test demonstrates it
 #[test]
 #[allow(clippy::cast_precision_loss)]
-fn time_trigger_execution_count_error_should_be_less_than_10_percent() -> Result<()> {
+fn time_trigger_execution_count_error_should_be_less_than_15_percent() -> Result<()> {
```

Comment:
Why do we need to relax requirement here?

Reply:
Because I didn't manage to get it within 10 % without eyeballing other constants for my machine. New-Sumeragi's block production time = a constant + cpu bound processing time + network latency. My understanding is that @Arjentix previously picked a block time estimate constant that had an error rate within 10 %.

Essentially, the "< 10 %" is not a guarantee made to the end customer/user. So I changed it to 15 % so that it would pass, and the end customer/user can get a faster consensus.

Reply:
Yeah, 10% was something that worked for me and for CI.
Although it's sad, that we need to relax the gap. Does Sumeragi works slower now?

Reply:
No. It's actually a lot faster. But I think the policy is slightly different. In the new Sumeragi, the leader waits for *block time* milliseconds before starting to create a block if the transaction cache isn't full. Then the whole networks policy is to commit the block as fast as possible. This means that when unloaded the networks total block time = [BLOCK_TIME_MS] + [leader: block validation runtime] + [worst latency to validating peer] + [validating peer: block validation runtime] + [worst latency to proxy tail] + [worst latency to nodes] + [wsv apply runtime]. This is supposed to be less than [BLOCK_TIME_MS] + [COMMIT_TIME_MS] otherwise the network is considered faulty.

And if the network is loaded, then we produce blocks as fast as possible, which means that the constant is inevitably wrong.

There is a fix however. We could in the future add code that calculates an estimated block time based on previous block times right? A rolling average of the previous X block times for instance.

Reply:
Yeah, @appetrosyan has also told about analysing previous blocks to predict the next block timestamp. That's our plan for a future